mappings/net/minecraft/world/biome/Biome.mapping

```diff
@@ -37,10 +57,10 @@ CLASS alv net/minecraft/world/biome/Biome
 	FIELD h WATER Lave;
 	FIELD i BIOMES Ljava/util/Set;
 	FIELD j NAMED_BIOME_IDS Leh;
-	FIELD m DECORATOR_DOUBLE_PLANT Layb;
-	FIELD n DECORATOR_TREE Lazi;
-	FIELD o DECORATOR_TREE_LARGE_OAK Laxs;
-	FIELD p DECORATOR_TREE_OAK Lazg;
```

Comment:
I changed it because it was inconsistant with the other things. Personally I would like them to just not have DECORATOR at all like blocks, but I figured this would be a good compromise.

If we change this back, should we change other static finals to have prefixes too?